roles/ciao-compute/tasks/main.yml

```diff
@@ -13,6 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+  - name: Copy CIAO Binaries
+    synchronize: src={{ item }} dest=/usr/bin
```

Comment:
I don't know if we would prefer to install non-official binaries on `/usr/local/bin` instead of `/usr/bin`, this way we wouldn't *override* the system binaries if they exists.
Also, as far as I can see,  `$PATH` is set to look for `/usr/loca/bin` first and then for `/usr/bin` (of course this can be changed by user) so really don't know...

Any input @obedmr ?